data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -2020,6 +2020,333 @@
     "//": "Police issue weapons of all types",
     "items": [ { "group": "longguns_cop", "prob": 20 }, { "group": "sidearms_cop", "prob": 80 } ]
   },
+  {
+    "type": "item_group",
+    "id": "police_carbines",
+    "//": "Carbines that US police would have",
+    "subtype": "distribution",
+    "entries": [ { "group": "modular_ar15", "prob": 100, "charges": [ 30, 30 ], "contents-group": "police_carbine_mods" } ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_carbines_damaged_extramags",
+    "//": "Carbines that US police would have, worn down and with used magazines",
+    "subtype": "collection",
+    "entries": [
+      { "group": "police_carbines", "prob": 100, "charges": [ 0, 30 ], "damage": [ 0, 3 ], "dirt": [ 0, 1000 ] },
+      { "group": "field_ar15_223" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_shotguns",
+    "//": "Shotguns that US police would have",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "mossberg_930", "variant": "m1014", "prob": 10, "charges": [ 0, 8 ], "contents-group": "police_shotgun_mods" },
+      { "item": "remington_870_express", "prob": 50, "charges": [ 0, 7 ], "contents-group": "police_shotgun_mods" },
+      { "item": "mossberg_500", "prob": 40, "charges": [ 0, 6 ], "contents-group": "police_shotgun_mods" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_pistols",
+    "//": "Pistols that US police would have",
+    "subtype": "distribution",
+    "entries": [
```

Comment:
m11, m17, m18, and p226 (mk 25) belong here. I would say STI 2011's (to stand in for staccato p's) but we only have them in 10mm.

Reply:
Added! Thank you.